lib/github_changelog_generator/options.rb

```diff
@@ -112,6 +112,11 @@ def add_sections?
       !self[:add_sections].nil? && !self[:add_sections].empty?
     end
 
+    # Boolean method for whether write to :output
```

Comment:
You can express this as a YARD docblock comment: `# @return [Boolean] whether to write to :output`

Reply:
Fix comment at 272a9a4